src/main/java/ch/njol/skript/effects/EffToggle.java

```diff
@@ -123,13 +133,23 @@ public boolean init(final Expression<?>[] vars, final int matchedPattern, final
 	
 	@Override
 	protected void execute(final Event e) {
+		Object[] values = expressions.getArray(e);
+		if (values[0] instanceof Boolean) {
```

Comment:
Because the first is a Boolean, doesn't mean the rest is, the second element could still be a Block